test/unittests/execute_call_test.cpp

```diff
@@ -58,6 +58,69 @@ TEST(execute_call, call_with_arguments)
     EXPECT_THAT(execute(parse(wasm), 1, {}), Result(4));
 }
 
+TEST(execute_call, call_void_with_zero_arguments)
+{
+    /* wat2wasm
+    (module
+      (global $z (mut i32) (i32.const -1))
+      (func $set
+        (global.set $z (i32.const 1))
+      )
+      (func (result i32)
+        call $set
+        global.get $z
+      )
+    )
+    */
+    const auto wasm = from_hex(
+        "0061736d010000000108026000006000017f03030200010606017f01417f0b0a0f020600410124000b06001000"
+        "23000b");
+
+    EXPECT_THAT(execute(parse(wasm), 1, {}), Result(1));
+}
+
+TEST(execute_call, call_void_with_one_argument)
```

Comment:
@chfast I think these tests could be merged anyway.

Reply:
#641.